.circleci/config.yml

```diff
@@ -16,39 +16,40 @@ jobs:
       - checkout
       - setup_remote_docker
       - run:
-          command: |
-            /scripts/build-image.sh web3f/cla-assistant .
+          command: docker build -t web3f/cla-assistant .
 
-  integrationTests:
+  publishImage:
     docker:
       - image: web3f/ci-commons:v1.7.1
     steps:
       - checkout
       - setup_remote_docker
       - run:
-          description: run integration tests
-          command: |
-            /scripts/integration-tests.sh
+          command: /scripts/publish-image.sh web3f/cla-assistant
 
-  publishImage:
+  testChart:
     docker:
       - image: web3f/ci-commons:v1.7.1
     steps:
       - checkout
-      - setup_remote_docker
       - run:
           command: |
-            /scripts/publish-image.sh web3f/cla-assistant
+            helm template ./charts/cla-assistant/.
+
+  integrationTests:
+    docker:
+      - image: web3f/ci-commons:v1.7.1
+    steps:
+      - checkout
```

Comment:
don't work yet, need to sort a few things before I include them.

Reply:
how does budapest-acipenser sound as a default name?

  testChart:
    docker:
      - image: web3f/ci-commons:v1.7.1
    steps:
      - checkout
      - run:
          command: |
            helm lint ./charts/cla-assistant/.
      - run:
          command: |
            helm install --dry-run --debug --generate-name ./charts/cla-assistant/.
      - run:
          command: |
            helm install budapest-acipenser ./charts/cla-assistant/.
      - run:
          command: |
            helm test budapest-acipenser

Reply:
Good name :) wrt the test steps, i think it would be better in the ci config just call `/scripts/integration-tests.sh` (the one from ci-commons image) and then do all the customization on `./scripts/integration-tests.sh` (the one in the repo) the same as we do in the other projects, this way we get kind setup for free.